### PR TITLE
Fixes various logging nitpicks

### PR DESCRIPTION
**Pull request by oleorhagen**
Some minor things which annoyed me when using Mender myself.

* Some log messages start with a lowercase, whilst others do not
* Wording in some places, uses words like we
* Minor wording choices

Changelog: Title

Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

**Comment by oleorhagen** [comment]
Really not much to it. Commit message says it all.

File: statescript/executor.go [hunk 1]
```diff
@@ -243,9 +243,9 @@ func execute(name string, timeout time.Duration) error {
 
 	if len(bts) > 0 {
 		if len(bts) > 10*1024 {
-			log.Errorf("stderr collected while running script %s [%s] (Truncated to 10KB)", name, bts[:10*1024])
+			log.Errorf("The standard-error collected while running script %s [%s] (Truncated to 10KB)", name, bts[:10*1024])
 		} else {
-			log.Errorf("stderr collected while running script %s [%s]", name, string(bts))
+			log.Errorf("The standard-error output collected while running script %s [%s]", name, string(bts))
```
Comment (lluiscampos) [inline statescript/executor.go]: To be consistent, remove "output" from the second case

File: statescript/executor.go [hunk 2]
```diff
@@ -79,7 +79,7 @@ func (l Launcher) getTimeout() time.Duration {
 	if l.Timeout != 0 {
 		return time.Duration(l.Timeout) * time.Second
 	}
-	log.Debugf("statescript: timeout for executing scripts is not defined; using default of %s seconds", defaultStateScriptTimeout)
+	log.Debugf("Statescript: The timeout for executing scripts is not defined; using default of %s seconds", defaultStateScriptTimeout)
```
Comment (lluiscampos) [inline statescript/executor.go]: I think these are weird. I would suggest using `statescript` (as it was), `StateScript` or `State script`.

File: statescript/executor.go [hunk 3]
```diff
@@ -243,9 +243,9 @@ func execute(name string, timeout time.Duration) error {
 
 	if len(bts) > 0 {
 		if len(bts) > 10*1024 {
-			log.Errorf("stderr collected while running script %s [%s] (Truncated to 10KB)", name, bts[:10*1024])
+			log.Errorf("The standard-error collected while running script %s [%s] (Truncated to 10KB)", name, bts[:10*1024])
```
Comment (kacf) [inline statescript/executor.go]: Usage of "The" here is weird, I think the original was better. But maybe a compromise? `Stderr messages collected...`

Comment (oleorhagen) [reply statescript/executor.go]: How about: `Collected stderr while running...` ?

Comment (kacf) [reply statescript/executor.go]: :+1:

**Comment by oleorhagen** [comment]
Latest pipeline along with integration/832: https://gitlab.com/Northern.tech/Mender/mender-qa/pipelines/126121288

**Pull request merged**